src/style.css

```diff
@@ -1 +1,91 @@
-/* Styles go here */
+* {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+.container {
+  box-sizing: border-box;
+  padding: 0;
+  line-height: 100%;
+  float: right;
+}
```

Comment:
do not `float` property it acts unexpectedly here 
you have already used  `box-sizing` and `padding` in `*` why are you using it here again
`line-height` wont work here